.github/workflows/main.yml

```diff
@@ -88,7 +90,7 @@ jobs:
                 echo 'false'
                 return
               else
-                input='${{ secrets.JDK_SUBMIT_PLATFORMS }}'
+                input='android-linux-aarch64, ios-macos-aarch64'
```

Comment:
Actually, this should be reverted, otherwise you can never run a user-specified set of platforms on GHA.